### PR TITLE
Make Carousel progress go up to 100%

### PR DESCRIPTION
**Pull request by hanbyul-here**
this is a nitpick, but it bothered me that the last progress bar doesn't reach up to 100%.

### CODE REVIEW

File: overrides/home/carousel/index.tsx [hunk 1]
```diff
@@ -38,7 +38,8 @@ export function DesktopCarousel () {
   }, []);
 
   const animationTimer = timer % slide_length;
-  const progressPercentage = Math.floor((animationTimer/slide_length) * 100)
+  // animationTimer/slide_length will never be 1, compensating the value here
+  const progressPercentage = Math.floor((animationTimer/slide_length) * (100*(slide_length/(slide_length-1))));
```
Comment (slesaad) [inline overrides/home/carousel/index.tsx:42]:
```suggestion
  const progressPercentage = Math.floor((animationTimer/slide_length-1) * 100);
```

Comment (hanbyul-here) [reply overrides/home/carousel/index.tsx:42]: urg yes! thank you!

Comment (slesaad) [reply overrides/home/carousel/index.tsx]: that broke the preview tho 😭 what happened???

Comment (hanbyul-here) [reply overrides/home/carousel/index.tsx]: Oh no T_T I guess mixed multiplication and division operations do not follow the commutative and associative laws...? (I looked this up) so `100* (slide_length/slide_length -1)` should be done separately, I guess?

Comment (hanbyul-here) [reply overrides/home/carousel/index.tsx]: I will revert the commit!

**Review by slesaad — APPROVED** [review]

**Pull request merged**